ACPs/77-reinventing-subnets/README.md

```diff
@@ -1,175 +1,180 @@
-- `disableOwner` is the only P-Chain owner allowed to disable the validator using `DisableSubnetValidatorTx`, specified below.
+- `remainingBalanceOwner` is the P-Chain owner where leftover $AVAX from the validator's Balance will be issued to when this validator it is removed from the validator set.
+- `disableOwner` is the only P-Chain owner allowed to disable the validator using `DisableL1ValidatorTx`, specified below.
 
-#### `SubnetValidatorRegistrationMessage`
+#### `L1ValidatorRegistrationMessage`
 
-The P-Chain can produce a `SubnetValidatorRegistrationMessage` for consumers to verify that a validation period has either begun or has been invalidated.
+The P-Chain can produce a `L1ValidatorRegistrationMessage` for consumers to verify that a validation period has either begun or has been invalidated.
 
-The `SubnetValidatorRegistrationMessage` is specified as an `AddressedCall` with `sourceChainID` set to the P-Chain ID, the `sourceAddress` set to an empty byte array, and a payload of:
+The `L1ValidatorRegistrationMessage` is specified as an `AddressedCall` with `sourceChainID` set to the P-Chain ID, the `sourceAddress` set to an empty byte array, and a payload of:
 
-| Field | Type | Size |
-| -: | -: | -: |
-| `codecID` | `uint16` | 2 bytes |
-| `typeID` | `uint32` | 4 bytes |
+|          Field |       Type |     Size |
+| -------------: | ---------: | -------: |
+|      `codecID` |   `uint16` |  2 bytes |
+|       `typeID` |   `uint32` |  4 bytes |
 | `validationID` | `[32]byte` | 32 bytes |
-| `registered` | `bool` | 1 byte  | 
-| | | 39 bytes |
+|   `registered` |     `bool` |   1 byte |
+|                |            | 39 bytes |
 
 - `codecID` is the codec version used to serialize the payload, and is hardcoded to `0x0000`
 - `typeID` is the payload type identifier and is `0x00000002` for this message
 - `validationID` identifies the validator for the message
 - `registered` is a boolean representing the status of the `validationID`. If true, the `validationID` corresponds to a validator in the current validator set. If false, the `validationID` does not correspond to a validator in the current validator set, and never will in the future.
 
-#### `SubnetValidatorWeightMessage`
+#### `L1ValidatorWeightMessage`
 
-The P-Chain can consume a `SubnetValidatorWeightMessage` through a `SetSubnetValidatorWeightTx` to update the weight of an existing validator. The P-Chain can also produce a `SubnetValidatorWeightMessage` for consumers to verify that the validator weight update has been effectuated.
+The P-Chain can consume a `L1ValidatorWeightMessage` through a `SetL1ValidatorWeightTx` to update the weight of an existing validator. The P-Chain can also produce a `L1ValidatorWeightMessage` for consumers to verify that the validator weight update has been effectuated.
```

Comment:
a `L1ValidatorWeightMessage` -> an `L1ValidatorWeightMessage`

(needs to be applied in a few places)